nearest_neighbor/nearest_neighbor.py

```diff
@@ -86,14 +86,10 @@ def get_definitions(wordList):
     # So, right now it adds all of the definitions for all words in the synsets.
     # Usually, the synsets just contain the same word but different defintions.
     # Occasionally they have synonyms that are different words.
-    definitions = {}
+    definitions = ""
```

Comment:
Let me know what you think of this. I'm never a huge fan of string concatenation, but for the model, it just needs a string with all of the definitions. If you can think of a better way to do this, I'd be happy to change it.

Reply:
I don't see a problem with this, I think your approach is great.